addons/backgrounds/README.md

```diff
@@ -45,7 +45,7 @@ You can add the backgrounds to all stories with `addDecorator` in `.storybook/co
 
 ```js
 import { addDecorator } from '@storybook/react'; // <- or your storybook framework
-import { withBackgrounds } from '@storybook/addon-backgrounds';
+import withBackgrounds from '@storybook/addon-backgrounds';
```

Comment:
Default export is deprecated. Why would you like to add it to README?

Reply:
I meant that the correct way to import the component is without the curly brackets. With the curly brackets are returning undefined

Reply:
What version are you using?

Reply:
The version is ^3.4.10

Reply:
And the README you are changing is for v4.alpha

Reply:
The installation instructions of this file is installing the version 3.4.10. 
I think that accept this change while the v4 is not on npm will prevent mistakes